rustls/src/crypto/aws_lc_rs/mod.rs

```diff
@@ -50,15 +50,12 @@ pub fn default_provider() -> CryptoProvider {
 }
 
 fn default_kx_groups() -> Vec<&'static dyn SupportedKxGroup> {
-    #[cfg(feature = "fips")]
-    {
-        DEFAULT_KX_GROUPS
-            .iter()
-            .filter(|cs| cs.fips())
-            .copied()
-            .collect()
-    }
-    #[cfg(not(feature = "fips"))]
+    // NOW REMOVED FROM THIS FORK:
+    // #[cfg(... "fips")]
+    // {
+    //     ...
+    // }
+    // NOW REMOVED FROM THIS FORK: #[cfg(not(... "fips"))]
     {
         DEFAULT_KX_GROUPS.to_vec()
     }
```

Comment:
I think this would be ideal, but don't think this update should be done:

```suggestion
    // NOW REMOVED FROM THIS FORK: #[cfg(not(... "fips"))]
    DEFAULT_KX_GROUPS.to_vec()
```

This update would both increase divergence from upstream rustls and likely increase the difficulty of restoring the fips feature if needed someday in the future.